src/frontends/pytorch/src/op_table.cpp

```diff
@@ -795,6 +795,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_ts() {
         {"prim::TupleIndex", op::translate_tuple_index},
         // prim::TupleUnpack - Supported in limited set of patterns
         {"prim::type", op::skip_node},  // Used with prim::device, pass PtFrameworkNode.
+        {"prim::data", op::translate_data},
```

Comment:
I think it can be `op::skip_node`. Doesn't think you need any special handling for complex types in this case.